CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
Though maybe we should use FALSE as the `none` case.
So the values are `FALSE`, `NOT-MERGED`, and `MERGED`?